package.json

```diff
@@ -28,6 +28,7 @@
     "bugsnag-react-native": "^2.9.3",
     "contentful": "^5.0.5",
     "core-js": "^2.5.6",
```

Comment:
Does this mean pulling in core-js can go? We added this in order to support some of the new String prototype methods.

Reply:
I've been trying to check this all week but could never get Android(on any branch) to run on my laptop. Trying on desktop now.